gcloud-java-storage/src/main/java/com/google/gcloud/storage/StorageImpl.java

```diff
@@ -577,6 +577,68 @@ public URL signUrl(BlobInfo blobInfo, long expiration, SignUrlOption... options)
     }
   }
 
+  @Override
+  public List<BlobInfo> get(BlobInfo blobInfo1, BlobInfo blobInfo2, BlobInfo... blobInfos) {
+    List<Tuple<StorageObject, Map<StorageRpc.Option, ?>>> toGet =
+        Lists.newArrayListWithCapacity(blobInfos.length + 2);
+    toGet.add(Tuple.<StorageObject, Map<StorageRpc.Option, ?>>of(blobInfo1.toPb(), optionMap()));
+    toGet.add(Tuple.<StorageObject, Map<StorageRpc.Option, ?>>of(blobInfo2.toPb(), optionMap()));
+    for (BlobInfo blobInfo : blobInfos) {
+      toGet.add(Tuple.<StorageObject, Map<StorageRpc.Option, ?>>of(blobInfo.toPb(), optionMap()));
+    }
+    StorageRpc.BatchResponse response =
```

Comment:
Though this is slightly more efficient I am not sure I favor it over using Batch on the service layer.